<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OTPMainWindow</class>
 <widget class="QMainWindow" name="OTPMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GGR OpenTripPlanner Plugin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Projekt</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Neues Projekt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Löschen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Klonen</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="start_orig_dest_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Berechnungen starten</string>
        </property>
        <property name="text">
         <string>Berechnung starten</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Layer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Startlayer</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="3" column="0">
            <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_2"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Wohnblöcke</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_3"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Ausgänge der Wohnblöcke</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="selected_only_check_2">
             <property name="text">
              <string>nur selektierte Punkte</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Ziellayer</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Eingänge zu den Grünflächen</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="selected_only_check_3">
             <property name="text">
              <string>nur selektierte Punkte</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Parameter</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Parameter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="6" column="4">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>m/s</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_17">
             <property name="toolTip">
              <string>maximale Distanz, die auf der Route zu Fuß zurückgelegt werden darf 
maximal einstellbarer Wert entspricht unendlich 
ACHTUNG: wird ignori</string>
             </property>
             <property name="text">
              <string>max. Laufdistanz</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_23">
             <property name="toolTip">
              <string>maximale Anzahl der Umstiege auf der Route</string>
             </property>
             <property name="text">
              <string>Puffer</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_33">
             <property name="toolTip">
              <string>Berücksichtung der Zugänglichkeit für Rollstühle an/aus</string>
             </property>
             <property name="text">
              <string>Zugänglichkeit Rollstuhl </string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_19">
             <property name="toolTip">
              <string>maximale Fahrradgeschwindigkeit</string>
             </property>
             <property name="text">
              <string>max. Fahrradgeschw.</string>
             </property>
            </widget>
           </item>
           <item row="10" column="3">
            <widget class="QDoubleSpinBox" name="max_slope_edit">
             <property name="toolTip">
              <string>maximale Steigung, bis zu der ein Routenabschnitt als für Rollstühle zugänglich gilt</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QLabel" name="label_21">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="max_walk_dist_edit_2">
             <property name="toolTip">
              <string>maximale Distanz, die auf der Route zu Fuß zurückgelegt werden darf 
maximal einstellbarer Wert entspricht unendlich 
ACHTUNG: wird ignori</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="maximum">
              <number>50000</number>
             </property>
             <property name="value">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_15">
             <property name="toolTip">
              <string>maximale Laufgeschwindigkeit</string>
             </property>
             <property name="text">
              <string>max. Laufgeschwindigkeit</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QDoubleSpinBox" name="bike_speed_edit">
             <property name="toolTip">
              <string>maximale Fahrradgeschwindigkeit</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_22">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>m²</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="label_18">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QSpinBox" name="transfers_edit">
             <property name="toolTip">
              <string>maximale Anzahl der Umstiege auf der Route</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_24">
             <property name="toolTip">
              <string>maximale Distanz, die auf der Route zu Fuß zurückgelegt werden darf 
maximal einstellbarer Wert entspricht unendlich 
ACHTUNG: wird ignori</string>
             </property>
             <property name="text">
              <string>benötigte Grünfläche je Einwohner</string>
             </property>
            </widget>
           </item>
           <item row="7" column="4">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>m/s</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="5">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QDoubleSpinBox" name="walk_speed_edit">
             <property name="toolTip">
              <string>maximale Laufgeschwindigkeit</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QCheckBox" name="wheelchair_check">
             <property name="toolTip">
              <string>Berücksichtung der Zugänglichkeit für Rollstühle an/aus</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="max_walk_dist_edit">
             <property name="toolTip">
              <string>maximale Distanz, die auf der Route zu Fuß zurückgelegt werden darf 
maximal einstellbarer Wert entspricht unendlich 
ACHTUNG: wird ignori</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="maximum">
              <number>50000</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_34">
             <property name="toolTip">
              <string>maximale Steigung, bis zu der ein Routenabschnitt als für Rollstühle zugänglich gilt</string>
             </property>
             <property name="text">
              <string>max. Steigung für Rollstühle</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>verwendetes Netz</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>Neu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Löschen</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="router_combo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="close_action"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="manual_action"/>
    <addaction name="separator"/>
    <addaction name="info_action"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="info_action">
   <property name="text">
    <string>Über</string>
   </property>
  </action>
  <action name="close_action">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
  <action name="load_config_action">
   <property name="text">
    <string>Laden</string>
   </property>
  </action>
  <action name="save_config_action">
   <property name="text">
    <string>Speichern unter</string>
   </property>
  </action>
  <action name="reset_config_action">
   <property name="text">
    <string>Auf defaults zurücksetzen</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="actiona_2">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="manual_action">
   <property name="text">
    <string>Anleitung</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
